presto-ui/src/components/LivePlan.jsx

```diff
@@ -250,7 +251,7 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
                                 class: "plan-edge",
                                 style: "stroke-width: 4px",
                                 arrowheadClass: "plan-arrowhead",
-                                label: sourceStats.outputDataSize + " / " + formatRows(sourceStats.outputPositions),
+                                label: sourceStats.outputDataSizeInBytes + " / " + formatRows(sourceStats.outputPositions),
```

Comment:
will this be less user friendly/readable then previously (e.g. would it previously show 10GB, and now will show 10000000000 bytes)? Can we convert it back to something that will be readable? Same comment for the rest of the usages

Reply:
`utils.formatDataSizeBytes()` could be used here to improve the readability